### PR TITLE
AIO-866 sitemap with site service

### PR DESCRIPTION
**Pull request by emilynielson**

### CODE REVIEW

File: blocks/sitemap-section-feature-block/features/sitemap-section/xml.js [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+  })
+}
+
+SitemapSection.propTypes = {
+  customFields: PropTypes.shape({
+    feedPath: PropTypes.string.tag({
+      label: 'Feed Path',
+      group: 'Format',
+      description: 'Relative URL path used in the generated link',
+      defaultValue: '/arc/outboundfeeds/sitemap/category',
```
Comment (tbrick855) [inline blocks/sitemap-section-feature-block/features/sitemap-section/xml.js]: default value should be ''

File: blocks/sitemap-section-feature-block/features/sitemap-section/xml.js [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+    }),
+    feedParam: PropTypes.string.tag({
+      label: 'URL Parameters',
+      group: 'Format',
+      description:
+        'Optional parameters to append to URL, join multiple parameters with &',
+      defaultValue: 'outputType=xml',
```
Comment (tbrick855) [inline blocks/sitemap-section-feature-block/features/sitemap-section/xml.js]: default value should be ''

File: blocks/sitemap-section-feature-block/features/sitemap-section/xml.js [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+        'Comma separated list of section IDs to exclude from the feed, i.e /subscribe,/test',
+      defaultValue: '',
+    }),
+    excludeLinks: PropTypes.bool.tag({
+      label: 'Exclude links',
+      group: 'Format',
+      description: 'Should links be excluded from the output',
+      defaultValue: true,
+    }),
+    priority: PropTypes.oneOf([
```
Comment (tbrick855) [inline blocks/sitemap-section-feature-block/features/sitemap-section/xml.js]: priority and changeFreq are part of the standard sitemap customFields.  Look at how videoSitemap uses generatePropsForFeed.  You will need to pass an array of customFields you want to exclude like 
['includePromo','promoItemsJmespath','resizerKVP','lastMod']

File: blocks/sitemap-section-feature-block/README.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# Sitemap-Section
+
+Sitemaps provide search engines with metadata regarding the specific news content on a website. Using the Sitemap, bots can quickly find the news articles contained on a site.
+This Sitemap identifies the url of every section in Site Service.
+
+This feed can be used to generate links to sitemaps like
+
+`http://wwww.example.com/arc/outboundfeeds/sitemap/category/{category}/`
```
Comment (tbrick855) [inline blocks/sitemap-section-feature-block/README.md]: Drop this example, just use sitemap to section

**Review by tbrick855 — CHANGES_REQUESTED** [review]
That was fast.  Nice work, We have a util to add common customFields.  Also I think this feeds default configs should default to just building links to their section fronts.  Links to sitemaps are only valid in sitemap-indexes.

File: blocks/sitemap-section-feature-block/features/sitemap-section/xml.js (hunk 1 above)
Comment (emilynielson) [reply blocks/sitemap-section-feature-block/features/sitemap-section/xml.js]: Makes sense! Updated

File: blocks/sitemap-section-feature-block/features/sitemap-section/xml.js (hunk 2 above)
Comment (emilynielson) [reply blocks/sitemap-section-feature-block/features/sitemap-section/xml.js]: Makes sense! Updated

File: blocks/sitemap-section-feature-block/features/sitemap-section/xml.js (hunk 3 above)
Comment (emilynielson) [reply blocks/sitemap-section-feature-block/features/sitemap-section/xml.js]: Ah ok! Sounds good! Updated

File: blocks/sitemap-section-feature-block/README.md (hunk 4 above)
Comment (emilynielson) [reply blocks/sitemap-section-feature-block/README.md]: Makes sense! Updated

**Review by tbrick855 — CHANGES_REQUESTED** [review]
You are missing the @wpmedia/feeds-prop-types dependency in your package.json
You have lint errors

**Comment by emilynielson** [comment]
My apologies, let me know if everything looks ok now

**Review by tbrick855 — APPROVED** [review]
LGTM

**Pull request merged**